### DISCUSSION TITLE
wonderign how retrieve data of the validation period.

### DISCUSSION THREAD

**Discussion by LucaPiciullo** [pastas/pastas#660] [Q&A] [answered]
hi there,

thanks for the nice product you have created.
I am trying to simulate volumetric water content in the soil as a function of many different climate varaibles. I have set a calibration period and a validation one. I am wonderign how can I retrieve the data of the validation period.
with:
ml.simulate()

I can get the data  until the Tmax period I have selected (i.e., calibration). I would like actually to have the valuse until the Tmax deifned in the plot.
any advice?

**Comment by mbakker7** [comment, marked as answer]
`ml.simulate` takes a `tmax` keyword, where you can define the end of the period that you want to simulate. By default the end is the end of calibration, but you can set it according to your wishes. Please make sure you have stress series that extend that long, otherwise you may get undesired results.

**Reply by LucaPiciullo** [reply]
thanks.
